src/Data/Managers/V2/POST/PostExpertStateTransfer/usePostExpertStateTransfer.ts

```diff
@@ -0,0 +1,65 @@
+import { UseMutationResult, useMutation } from "@tanstack/react-query";
+import { ParsedNumber } from "@/Core";
+import { PrimaryBaseUrlManager } from "@/UI";
+
+import { useFetchHelpers } from "../../helpers";
+
+/**
+ * Required attributes to construct the post request to force update the state of an instance in Expert mode
+ */
+export interface PostExpertStateTransfer {
+  message: string;
+  current_version: ParsedNumber;
+  target_state: string;
+  operation?: string;
+}
+
+/**
+ * React Query to Force update the state of an instance in expert mode.
+ *
+ * @param {string} environment - The Environment where the instance is located
+ * @param {string} instance_id - The hashed id of the instance
```

Comment:
it is hashed? I just don't know and I'm surprised to see that here

Reply:
I will change it to UUID, but yes, a UUID is hashed.